src/DynamoRevit/ViewModel/RevitWatch3DViewModel.cs

```diff
@@ -221,7 +223,10 @@ private IEnumerable<GeometryObject> Tessellate(Curve curve)
             {
                 var xyz0 = new XYZ(verts[i] * conv, verts[i + 1] * conv, verts[i + 2] * conv);
                 var xyz1 = new XYZ(verts[i + 3] * conv, verts[i + 4] * conv, verts[i + 5] * conv);
-
+                if (xyz0.DistanceTo(xyz1) < DocumentManager.Instance.CurrentUIApplication.Application.ShortCurveTolerance)
+                {
+                    continue;
+                }
```

Comment:
If Revit Polylines can have segments shorter than the tolerance, then discarding initial lines below that tolerance should avoid that problem... correct?